src/pages/similarity-indexes.html

```diff
@@ -43,47 +43,47 @@ <h3 class="mb-0"><span class="collapsible-heading" data-toggle="collapse" data-t
                         class="index-row"
 						ng-class="{'table-info text-secondary':index.name == selected.name}">
 						<td>
-                            <a class="lead index-name" ng-href="{{(index.status == 'BUILT' || index.status == 'REBUILDING' || index.status == 'OUTDATED') ? '#' : ''}}" ng-click="goToSimilarityIndex(index)">{{index.name}}</a>
+                            <a class="lead index-name" ng-href="{{(similarityIndexStatus.BUILT === index.status || similarityIndexStatus.REBUILDING === index.status || similarityIndexStatus.OUTDATED === index.status) ? '#' : ''}}" ng-click="goToSimilarityIndex(index)">{{index.name}}</a>
 						</td>
 						<td>
-							<span class="tag tag-default" ng-show="index.type == 'text'" gdb-tooltip="{{'index.type.tooltip' | translate}}">{{'text.type.index' | translate}}</span>
-							<span class="tag tag-default" ng-show="index.type == 'predication'" gdb-tooltip="{{'index.type.tooltip' | translate}}">{{'predication.type.index' | translate}}</span>
-							<span class="tag tag-default" ng-show="index.type == 'textLiteral'" gdb-tooltip="{{'index.type.tooltip' | translate}}">{{'literal.type.index' | translate}}</span>
+							<span class="tag tag-default" ng-show="similarityIndexType.TEXT === index.type" gdb-tooltip="{{'index.type.tooltip' | translate}}">{{'text.type.index' | translate}}</span>
```

Comment:
Maybe change to ng-if where possible

Reply:
I tried with copy-paste but there are failed tests. I will do this in separate MR.